unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/FriendsController.cs

```diff
@@ -93,22 +94,49 @@ public async UniTask<FriendRequest> RequestFriendshipAsync(string friendUserId,
 
             friendRequests[friendRequest.FriendRequestId] = friendRequest;
 
+            UpdateFriendshipStatus(new FriendshipUpdateStatusMessage
+                { action = FriendshipAction.REQUESTED_TO, userId = friendUserId });
+
             return friendRequest;
         }
 
-        public void RequestFriendship(string friendUserId)
-        {
+        public void RequestFriendship(string friendUserId) =>
             apiBridge.RequestFriendship(friendUserId);
-        }
 
-        public Dictionary<string, UserStatus> GetAllocatedFriends()
+        public Dictionary<string, UserStatus> GetAllocatedFriends() =>
+            new Dictionary<string, UserStatus>(friends);
+
+        public async UniTask<FriendRequest> AcceptFriendshipAsync(string friendRequestId)
```

Comment:
The lack of cancellation support is intended? just curious

Reply:
the new rpc protocol does not support cancellation tokens yet